integreat_cms/api/tests/api_tests.py

```diff
@@ -0,0 +1,64 @@
+from django.test import TestCase
+from linkcheck.listeners import disable_listeners
+from .api_tests_utils import generate_test_functions
+
+endpoint_data = [
+    (
```

Comment:
```suggestion
    (
        "/api/regions/",
        "/wp-json/extensions/v3/sites/",
        "integreat_cms/api/tests/expected-outputs/regions.json",
    ),
    (
```
You already have the expected json for all regions, so why not just test it as well?

Reply:
Testing API all region method has a problem:
The test always fails because the http status code is 301 permanently moved
But we can test it easily when dropping the status 200 check
as we compare the json response anyways, imo we can drop the status check

Reply:
Alternatively we could allow 200 and 301 as assertions

Reply:
Are you sure you didn't just forget a trailing slash? Then, the 301 is expected... we can always directly check the url where the 301 redirect is pointing to...

Reply:
Indeed, the trailing slash seems to fix the 301 code, I didn't now that
Thank you :smile: